app/components/cloud-call.js

```diff
@@ -110,7 +94,6 @@ export default Component.extend({
       if (!shouldRender) return;
```

Comment:
yeah I was lazy when I first dumped this into the app.  i should have ember-ized it from the get go.  Ember methods work great with ThreeJS but u gotta know some ember trix to really nail it